pkg/api/flow.go

```diff
@@ -139,8 +141,6 @@ func (h *flowHandler) initRoutes(r *mux.Router) {
 	//       "$ref": '#/definitions/ErrorResponse'
 	r.HandleFunc("/namespaces/{ns}", h.CreateNamespace).Name(RN_AddNamespace).Methods(http.MethodPut)
 
-	r.HandleFunc("/namespaces/{ns}/lint", h.NamespaceLint).Name(RN_GetNamespaceLogs).Methods(http.MethodGet)
```

Comment:
We need to verify if this is safe to remove

Reply:
@stefan-kracht, do we use this `lint` Endpoint ?

Reply:
Yes. We use it here:
![CleanShot 2024-03-21 at 11 03 16@2x](https://github.com/direktiv/direktiv/assets/121789579/631fe014-161d-4ce0-8649-a9c68d872347)